app/products/calls/components/calls_custom_message/calls_custom_message.tsx

```diff
@@ -41,61 +40,74 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
             flexDirection: 'row',
             alignItems: 'center',
             color: changeOpacity(theme.centerChannelColor, 0.6),
-            paddingTop: 5,
-            paddingBottom: 5,
+            padding: 12,
+            marginBottom: 2,
+            gap: 8,
+            backgroundColor: theme.centerChannelBg,
+            borderWidth: 1,
+            borderColor: changeOpacity(theme.centerChannelColor, 0.12),
+            borderRadius: 4,
+            shadowColor: theme.centerChannelColor,
+            shadowOffset: {
+                width: 0,
+                height: 2,
+            },
+            shadowRadius: 3,
```

Comment:
Not sure why, but `shadowRadius: 1` looks closer to Figma. Maybe the react native values don't map properly or something